src/dndbeyond/base/utils.js

```diff
@@ -264,8 +264,9 @@ async function buildAttackRoll(character, attack_source, name, description, prop
                     } else {
                         // Apply great weapon fighting to brutal damage dice
                         if ((character.hasClassFeature("Great Weapon Fighting", true) || character.hasFeat("Great Weapon Fighting", true)) &&
-                            properties["Attack Type"] == "Melee" &&
-                            ((properties["Properties"].includes("Versatile") && character.getSetting("versatile-choice") != "one") || properties["Properties"].includes("Two-Handed"))) {
+                            (properties["Attack Type"] == "Melee" &&
+                            ((properties["Properties"].includes("Versatile") && character.getSetting("versatile-choice") != "one") || properties["Properties"].includes("Two-Handed")) ||
+                            name == "Polearm Master - Bonus Attack")) {
```

Comment:
I think the `||` should be before that last parenthesis instead of after it. 
`if (GWF && (melee && ((versatile && not-one-handed) || two-handed) || polearm-master)`

You basically have a `A && B || C` (`melee && versatile-stuff || polearm-master`) which is unreliable/confusing in how it would behave. It should be `(A && B) || C` or `A && (B || C)`

Reply:
Fixed.

I blame the horrifying line breaks in that if statement 😅

Reply:
Yeah, it nearly broke my brain